packages/web-components/src/components/va-on-this-page/va-on-this-page.css

```diff
@@ -7,43 +7,49 @@
   border-radius: 4px;
   padding: 1em;
 }
+
 @media (min-width: 768px) {
   :host {
     border: none;
     padding: 1em 1em 1em 0;
   }
 }
 
-dt {
+#on-this-page {
   font-family: var(--font-serif);
   font-size: 1.25rem;
   margin: 0.4em 0em;
 }
 
-dd {
-  margin: 0.4em 0em;
-}
-
-dl {
+ul {
   margin-top: 0;
   margin-bottom: 0;
   margin-left: 0;
   list-style-type: none;
   padding-left: 0;
 }
 
-dd a {
-  display: flex;
-  align-items: baseline;
+li {
   padding: 0.5em;
+  max-width: 372px;
```

Comment:
@Andrew565 Would you be able to revert the width back to the original? My apologies for adding that feedback to this ticket. I talked to Carol about the issue she was able to move the ticket ([#951](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/951)) to address the width to Sprint 13 so we can address the width then.

Reply:
Done